{"stats": {"errors": 0, "failures": 2, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 2}, "results": [{"classname": "index_test:TxTest", "name": "test_coinbase_height", "time": 0.00047516822814941406, "type": "failure", "errtype": "builtins.AssertionError", "message": "None != 465879", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/adamjonas/coinbases-jsong-pb-sf-072618/test/index_test.py\", line 16, in test_coinbase_height\n    self.assertEqual(tx.coinbase_height(), 465879)\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 829, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 822, in _baseAssertEqual\n    raise self.failureException(msg)\nAssertionError: None != 465879\n"}, {"classname": "index_test:TxTest", "name": "test_is_coinbase", "time": 0.00020170211791992188, "type": "failure", "errtype": "builtins.AssertionError", "message": "False is not true", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/adamjonas/coinbases-jsong-pb-sf-072618/test/index_test.py\", line 10, in test_is_coinbase\n    self.assertTrue(tx.is_coinbase())\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 682, in assertTrue\n    raise self.failureException(msg)\nAssertionError: False is not true\n"}]}